### PR TITLE
Use generated editor tool for Xtext

### PR DESCRIPTION
**Pull request by szschaler**
This example uses the capabilities defined in https://github.com/mdenet/platformtools/pull/36

### CODE REVIEW

**Comment by szschaler** [comment]
Only merge once https://github.com/mdenet/platformtools/pull/36 has been merged.

File: xtext-turtles/xtext_activity.json [hunk 1]
```diff
@@ -81,52 +81,85 @@
...
+            {
+                "id": "panel-mm-composite",
+                "name": "The meta-model generated by Xtext",
+                "ref": "composite",
+                "childPanels": [
+                    {
+                        "id": "panel-mm",
+                        "name": "XMI text format",
+                        "ref": "ecore",
+                        "file": "{{ID-panel-turtles}}/xtext-resources/generated/meta-model.ecore"
```
Comment (barnettwilliam) [inline xtext-turtles/xtext_activity.json:106]: General note, the path where generated resources are available should be documented.

**Review by barnettwilliam — APPROVED** [review]
LGTM

File: xtext-turtles/xtext_activity.json (hunk 1 above)
Comment (szschaler) [reply xtext-turtles/xtext_activity.json:106]: Done in https://github.com/mdenet/platformtools/pull/36 via a tool-specific readme file

**Pull request merged**